### PR TITLE
Improved what is conda forge section.

### PR DESCRIPTION
**Pull request by chicken-biryani**
This Pull request aims at fixing #1645
The docs earlier suggested that the only way of contributing to conda-forge was by contributing to packages. This PR tries to improve that.

PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [X] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by chicken-biryani** [comment]
@beckermr @kathatherine can you please have a look.

To be honest, I was extremely tempted to work on https://github.com/conda-forge/conda-forge.github.io/issues/1282 all at once. But I think baby-steps and smaller PRs are a better way to approach the problem.

**Comment by beckermr** [comment]
Looks pretty good!

File: src/user/introduction.rst [hunk 1]
```diff
@@ -9,14 +9,11 @@ A brief introduction
...
-
-**Cannot find a package or only outdated versions of a package?** - Everybody is welcome to contribute to our package stack! Please refer to :ref:`becoming_involved`, for an overview on how to start contributing.
+**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you! You can `search <https://anaconda.org/>`__ for packages online as well as keep an eye out for packages from our ``conda-forge`` organization.
 
+We value all kinds of contributions- not just code. A few recommended ways to start contributing to conda-forge are– `Contribute new packages <https://conda-forge.org/docs/maintainer/adding_pkgs.html>`_, Help update and `maintain packages <https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-pkgs>`_, Suggest or implement improvements for our `infrastructure <https://conda-forge.org/docs/maintainer/infrastructure.html#infrastructure>`_, and/or Help `improve the documentation <https://conda-forge.org/docs/user/contributing.html#improve-docs>`_.. For a detailed overview please refer to  :ref:`becoming_involved`
```
Comment (jaimergp) [inline src/user/introduction.rst:16]: Maybe this series of links reads better as a bullet point list? As in:


```suggestion
We value all kinds of contributions - not just code. A few recommended ways to start contributing to conda-forge are:

 - `Contribute new packages <https://conda-forge.org/docs/maintainer/adding_pkgs.html>`_
 - Help update and `maintain packages <https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-pkgs>`_
 - Suggest or implement improvements for our `infrastructure <https://conda-forge.org/docs/maintainer/infrastructure.html#infrastructure>`_
 - Help `improve the documentation <https://conda-forge.org/docs/user/contributing.html#improve-docs>`_.
 
 For a detailed overview please refer to  :ref:`becoming_involved`.
```

Comment (chicken-biryani) [reply src/user/introduction.rst:16]: Yes. I guess.

Also, another question- may sound silly... But I'll play my newbie card and take my chances. 😅

I believe creating more/maintaining recipes is a contribution as well?

Although it's not in the docs yet... Maybe the person can be redirected to staged recipes?

Comment (chicken-biryani) [reply src/user/introduction.rst:16]: Yes. I guess.

Also, another question- may sound silly... But I'll play my newbie card and take my chances. 😅

I believe creating more/maintaining recipes is a contribution as well?

Although it's not in the docs yet... Maybe the person can be redirected to staged recipes?

**Comment by jaimergp** [comment]
A part of that is mentioned in the "Contribute new packages" link, but maybe that's too focused on _new_ additions. We can add one more bullet point reflecting the need of [maintainers](https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role) in orphaned feedstocks. Finding orphaned feedstocks is a different issue, though. One can search for open PRs across conda-forge, and sort by age (as in this [query](https://github.com/pulls?q=is%3Aopen+is%3Apr+org%3Aconda-forge+archived%3Afalse+user%3Aconda-forge+sort%3Acreated-asc)), but it's not perfect and _very noisy_. Maybe somebody else from Core wants to chime in.

**Comment by BastianZim** [comment]
> Finding orphaned feedstocks is a different issue, though.

Might make sense to revive https://github.com/conda-forge/cfep/blob/main/cfep-10.md

**Comment by chicken-biryani** [comment]
@BastainZim

I have a few questions now... Again.

1) **Can't the process be automated?*
The way we add the 'stale bot'. If a PR/Issue has not been active for over say 30 days.... We close it. In a similar way for a recipe.

2) **Out of context but** -- while reading a blog, one of the blogs suggested addressing 'bad recipes' as a good small starter contribution.

**What's a 'bad' recipe? Are there any parameters to define a 'bad' recipe?**


3)** : Follow up on 2:**
In a blog it's said--

```Building a conda package requires a recipe. A conda-build recipe is a flat directory that contains the following files:

meta.yaml---A file that contains all the metadata in the recipe. Only package/name and package/version are required.

build.sh---The script that installs the files for the package on macOS and Linux. It is executed using the bash command.

bld.bat---The build script that installs the files for the package on Windows. It is executed using cmd.

run_test.[py,pl,sh,bat]---An optional Python test file, a test script that runs automatically if it is part of the recipe.```

I am not aware of the difference between conda-build and conda-forge,

BUT can the same parameters be there for conda-forge recipes? Or listed somewhere so that a new contributor can follow them?

**Comment by BastianZim** [comment]
> I have a few questions now... Again.

That's what we're here for! :)

Let me try to answer your questions in a list to separate them.

1. Which process are you talking about that should be automated exactly? If you mean finding outdated feedstocks then yes, that can be done. The missing part here is currently the CFEP (Conda-forge enhancement proposal) which needs to be accepted by the core team first.
2. A bad recipe can be many things but generally speaking, it's a recipe that isn't following best practises. These can be:
    1. Dependencies are not correctly specified. Either incorrect dependencies or incorrect versions
    2. The meta.yaml follows an old style that conda doesn't use anymore
    3. The license is not correct
    4. The package is not `noarch` but could be (That wastes CI resources)
    5. ...
    
    The different parts do not necessarily constitute a good/bad recipe. For example, a _build.sh_ and _bld.bat_ script are not always needed. Most python based recipes just specify a `script: {{ PYTHON }} -m pip install . -vv` under `build` which is sufficient. The only time a separate file is normally used is when the `script` requires multiple steps. Same goes for the _run_test_ file.
3. conda-build is what actually builds the package from the recipe and conda-forge is an organisation that offers a standardised and automated environment in which conda-build can be run. If you look at a feedstock, you will actually find that the CI [invokes](https://github.com/conda-forge/azure-quantum-feedstock/blob/bd9fb5dc10aad0d67f8cc2d4e6bfd6b9351eec33/.scripts/build_steps.sh#L60) conda-build to build the recipe (Although we are using mambabuild nowadays which uses conda-build but with the mamba solver which speeds things up) so they are not competing tools. Conda-forge is just an organisation that standardises a lot of things (Compilers, the way recipes are built, uploading of the package, etc.) and provides a lot of tooling so that it is easier to add a package. Otherwise all of the work that the CI currently does would have to be done manually by each uploader.

File: src/user/introduction.rst [hunk 2]
```diff
@@ -9,14 +9,18 @@ A brief introduction
...
-Conda-forge is a community effort that provides conda packages for a wide range of software.
+``conda-forge`` is a community effort that provides conda packages for a wide range of software.
 
-**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you!
+**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you! You can `search <https://anaconda.org/>`__ for packages online as well as keep an eye out for packages from our ``conda-forge`` organization.
 
-You can `search <https://anaconda.org/>`__ for packages online. Look out for packages provided by our ``conda-forge`` organization.
-
-**Cannot find a package or only outdated versions of a package?** - Everybody is welcome to contribute to our package stack! Please refer to :ref:`becoming_involved`, for an overview on how to start contributing.
+We value all kinds of contributions - not just code. A few recommended ways to start contributing to conda-forge are:
```
Comment (ngam) [inline src/user/introduction.rst:16]:
```suggestion
We value all kinds of contributions — not just code. A few recommended ways to start contributing to ``conda-forge`` are:
```

The dash here should be an em dash (—) not a single dash. Or you can go for three separate dashes (---)

**Comment by jaimergp** [comment]
Moved to index.rst and remixed with https://github.com/conda-forge/conda-forge.github.io/pull/1699

**Pull request merged**